examples/private_endpoint_service/README.md

```diff
@@ -0,0 +1,214 @@
+couchbase-capella_private_endpoint_service.new_service: Still creating... [3m20s elapsed]
+couchbase-capella_private_endpoint_service.new_service: Still creating... [3m30s elapsed]
+couchbase-capella_private_endpoint_service.new_service: Still creating... [3m40s elapsed]
+couchbase-capella_private_endpoint_service.new_service: Still creating... [3m50s elapsed]
+couchbase-capella_private_endpoint_service.new_service: Still creating... [4m0s elapsed]
+couchbase-capella_private_endpoint_service.new_service: Still creating... [4m10s elapsed]
+couchbase-capella_private_endpoint_service.new_service: Still creating... [4m20s elapsed]
+couchbase-capella_private_endpoint_service.new_service: Still creating... [4m30s elapsed]
+couchbase-capella_private_endpoint_service.new_service: Still creating... [4m40s elapsed]
+couchbase-capella_private_endpoint_service.new_service: Still creating... [4m50s elapsed]
+couchbase-capella_private_endpoint_service.new_service: Still creating... [5m0s elapsed]
+couchbase-capella_private_endpoint_service.new_service: Still creating... [5m10s elapsed]
+couchbase-capella_private_endpoint_service.new_service: Still creating... [5m20s elapsed]
+couchbase-capella_private_endpoint_service.new_service: Still creating... [5m30s elapsed]
+couchbase-capella_private_endpoint_service.new_service: Still creating... [5m40s elapsed]
+couchbase-capella_private_endpoint_service.new_service: Still creating... [5m50s elapsed]
+couchbase-capella_private_endpoint_service.new_service: Still creating... [6m0s elapsed]
+couchbase-capella_private_endpoint_service.new_service: Still creating... [6m10s elapsed]
+couchbase-capella_private_endpoint_service.new_service: Still creating... [6m20s elapsed]
+couchbase-capella_private_endpoint_service.new_service: Still creating... [6m30s elapsed]
+couchbase-capella_private_endpoint_service.new_service: Still creating... [6m40s elapsed]
+couchbase-capella_private_endpoint_service.new_service: Still creating... [6m50s elapsed]
+couchbase-capella_private_endpoint_service.new_service: Still creating... [7m0s elapsed]
+couchbase-capella_private_endpoint_service.new_service: Creation complete after 7m0s
+
+Apply complete! Resources: 1 added, 0 changed, 0 destroyed.
+```
+
+## LIST
+
+Command: `terraform plan`
+
+Sample Output:
+```
+terraform plan
+╷
+│ Warning: Provider development overrides are in effect
+│
+│ The following provider development overrides are set in the CLI configuration:
+│  - couchbasecloud/couchbase-capella in /Users/$USER/GolandProjects/terraform-provider-couchbase-capella/bin
+│
+│ The behavior may therefore not match any released version of the provider and applying changes may cause the state to become
+│ incompatible with published releases.
+╵
+data.couchbase-capella_private_endpoint_service.service_stats: Reading...
+data.couchbase-capella_private_endpoint_service.service_stats: Read complete after 0s
+
+Changes to Outputs:
+  + existing_auditlogexport = {
```

Comment:
"I guess the name is misspelled here

Reply:
good catch.  it's also spelled on line 122

Reply:
addressed in 3f82603